### PR TITLE
upcoming: [M3-6538] - Add Interaction Tokens, Minimally Cleanup Theme Files

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description
We're starting to define how to we want to cleanup the theme files with some preliminary work.
- Global tokens are theme agnostic
- Alias tokens are theme sensitive and requires types to be merged. 
    - This also means that alias tokens can be used anywhere in the app and will inherit light/dark values.

## Changes  🔄
- Removed `theme.bg.bgAccessRow` for token `theme.interactionTokens.Background.Secondary`
- Charts are the same for light/dark so simplified our Theme/ThemeOptions for now
    - Renamed `charts` to `chartTokens` 
- Added alias `interactionTokens` 

## Target release date 🗓️
N/A

## Preview 📷
No visual changes. `theme.bg.bgAccessRow` and `theme.interactionTokens.Background.Secondary` share the same underlying value of `#f7f7fa`

### Verification steps
- Ensure everything builds properly
- Ensure the value for `theme.interactionTokens.Background.Secondary` works in both light/dark mode
    - See the linode details page

Screenshot 2024-10-09 at 4 01 08 PM

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/foundations/themes/dark.ts [hunk 1]
```diff
@@ -38,7 +37,6 @@ export const customDarkModeOptions = {
   bg: {
     app: Color.Neutrals[100],
     appBar: tempReplacementforColorNeutralsBlack,
-    bgAccessRow: Color.Neutrals[80],
```
Comment (jaalah-akamai) [inline packages/manager/src/foundations/themes/dark.ts:41]: Now using token directly in app level code.

File: packages/manager/src/foundations/themes/dark.ts [hunk 2]
```diff
@@ -58,7 +56,6 @@ export const customDarkModeOptions = {
     borderTypography: Color.Neutrals[80],
     divider: Color.Neutrals[80],
   },
-  charts: { ...Chart },
```
Comment (jaalah-akamai) [inline packages/manager/src/foundations/themes/dark.ts:61]: No need for dark mode overrides for charts at the moment. They're the same for light/dark, so let's just inherit from the light theme.

File: packages/manager/src/foundations/themes/index.ts [hunk 3]
```diff
@@ -23,9 +26,7 @@ import type {
 
 export type ThemeName = 'dark' | 'light';
 
-type ChartLightTypes = typeof ChartLight;
-type ChartDarkTypes = typeof ChartDark;
-type ChartTypes = MergeTypes<ChartLightTypes, ChartDarkTypes>;
```
Comment (jaalah-akamai) [inline packages/manager/src/foundations/themes/index.ts:28]: Once again, these are the same so let's just use our light types.

File: packages/manager/src/features/Linodes/LinodeEntityDetail.styles.ts [hunk 4]
```diff
@@ -180,7 +180,7 @@ export const StyledCopyTooltip = styled(CopyTooltip, {
 export const StyledGradientDiv = styled('div', { label: 'StyledGradientDiv' })(
   ({ theme }) => ({
     '&:after': {
-      backgroundImage: `linear-gradient(to right,  ${theme.bg.bgAccessRowTransparentGradient}, ${theme.bg.bgAccessRow});`,
+      backgroundImage: `linear-gradient(to right,  ${theme.bg.bgAccessRowTransparentGradient}, ${theme.interactionTokens.Background.Secondary});`,
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Linodes/LinodeEntityDetail.styles.ts:183]: I'm not sure if this is just a skill-issue on my part, but I can't tell what this `backgroundImage` style is doing. If I remove it, I can't tell a difference in the UI

**Review by bnussman-akamai — APPROVED** [review]

File: packages/manager/src/features/Linodes/LinodeEntityDetail.styles.ts (hunk 4 above)
Comment (jaalah-akamai) [reply packages/manager/src/features/Linodes/LinodeEntityDetail.styles.ts:183]: It adds the most subtle gradient you've ever seen lol. I should just remove it.

Comment (jaalah-akamai) [reply packages/manager/src/features/Linodes/LinodeEntityDetail.styles.ts:183]: Although I'll keep it for now, I plan on updating it once we get some gradient tokens

**Review by hkhalil-akamai — APPROVED** [review]
Verified no visual regressions in the app.

**Pull request merged**